service/src/test/java/uk/gov/hmcts/reform/fpl/service/CaseRoleServiceTest.java

```diff
@@ -206,7 +207,9 @@ void shouldThrowsExceptionWhenLocalAuthorityUsersCannotBeFetch() {
 
         @Test
         void shouldGrantAccessToAllUsersWithOrganisationId() {
-            final Set<String> localAuthorityUsers = Set.of(USER_1_ID, USER_2_ID);
+            final Set<String> localAuthorityUsers = new TreeSet<>();
+            localAuthorityUsers.add(USER_1_ID);
+            localAuthorityUsers.add(USER_2_ID);
```

Comment:
Can be one lined if you want
```suggestion
            final Set<String> localAuthorityUsers = new TreeSet<>(List.of(USER_1_ID, USER_2_ID));
```